code/components/jomjol_wlan/read_wlanini.cpp

```diff
@@ -38,14 +37,16 @@ std::vector<string> ZerlegeZeileWLAN(std::string input, std::string _delimiter =
 
 
 
-void LoadWlanFromFile(std::string fn, char *&_ssid, char *&_password, char *&_hostname, char *&_ipadr, char *&_gw,  char *&_netmask, char *&_dns)
+void LoadWlanFromFile(std::string fn, char *&_ssid, char *&_password, char *&_hostname, char *&_ipadr, char *&_gw,  char *&_netmask, char *&_dns, char *&_http_username, char *&_http_password)
 {
     std::string ssid = "";
     std::string passphrase = "";
     std::string ipaddress = "";
     std::string gw = "";
     std::string netmask = "";
     std::string dns = "";
+    std::string http_username = "";
```

Comment:
Those initializations are redundant and may be actually slower than just saying `std::string http_username;` for unoptimized code. `std::string` is a proper class with a reasonable default constructor that does what you would expect: creates an empty string: https://en.cppreference.com/w/cpp/string/basic_string/basic_string

I like to be explicit about initializations nonetheless and would write this like this:
```C++
std::string http_username{};
```

However, for this PR it makes sense to adopt the style of the existing code. So this is mainly an FYI.